scripts/format_backend.sh

```diff
@@ -3,6 +3,6 @@
 cd "$(dirname ${0})/.."
 
 echo "Formatting the backend... Our style follows the Black code style."
-python -m black gradio test
-python -m isort --profile=black gradio test
-python -m flake8 --ignore=E731,E501,E722,W503,E126,E203,F403 gradio test --exclude gradio/__init__.py
+python -m black gradio test client/python/gradio_client
+python -m isort --profile=black gradio test client/python/gradio_client
+python -m flake8 --ignore=E731,E501,E722,W503,E126,E203,F403 gradio test client/python/gradio_client --exclude gradio/__init__.py client/python/gradio_client/__init__.py
```

Comment:
I think we need an `--exclude` for each file?

This script isn't working for me

<img width="701" alt="image" src="https://user-images.githubusercontent.com/41651716/227366220-c7ed26a2-f7c9-48b4-9bb7-5fd6afe63b84.png">

Reply:
Fixed. Turns out you can use comma (no space) to separate multiple filenames